dask/array/backends.py

```diff
@@ -231,6 +231,15 @@ def _concatenate(L, axis=0):
     tensordot_lookup.register(scipy.sparse.spmatrix, _tensordot_scipy_sparse)
     take_lookup.register(scipy.sparse.spmatrix, sparse_take)
 
+    try:
```

Comment:
Can you use the register_layz function here? That will catch import errors for you (ideally we would do this above as well, but feel free to ignore that refactor in this pr)